default.xml

```diff
@@ -30,15 +30,15 @@
   <project name="projects_libs.git" path="projects/projects_libs" remote="sel4devkit" revision="master" upstream="master" dest-branch="master"/>
   <project name="pruner.git" path="tools/pruner" revision="6df378e8d91856d204e4d024796b91c6802d0501" upstream="master" dest-branch="master"/>
   <project name="rumprun.git" path="tools/rumprun" revision="04464ab931c28ca1c78e807741d82a4792660183" upstream="master" dest-branch="master"/>
-  <project name="seL4.git" path="kernel" remote="sel4devkit" revision="master" upstream="master" dest-branch="master"/>
+  <project name="seL4.git" path="kernel" remote="seL4" revision="ae7798e3193a223de5c841bdf6ad319937b3403b" upstream="master" dest-branch="master"/>
   <project name="seL4_libs.git" path="projects/seL4_libs" revision="d8ae2c44e92d1bf19891344eb038752565f0b30e" upstream="master" dest-branch="master"/>
   <project name="seL4_projects_libs.git" path="projects/seL4_projects_libs" revision="b687e3d503ed685634713eae868b220bdcda8261" upstream="master" dest-branch="master"/>
-  <project name="seL4_tools.git" path="tools/seL4" remote="sel4devkit" revision="master" upstream="master" dest-branch="master">
+  <project name="seL4_tools.git" path="tools/seL4" remote="seL4" revision="2f15ba2dc6a08c30b74c722b0673131d2aab32ac" upstream="master" dest-branch="master">
     <linkfile src="cmake-tool/init-build.sh" dest="init-build.sh"/>
     <linkfile src="cmake-tool/griddle" dest="griddle"/>
   </project>
   <project name="sel4runtime" path="projects/sel4runtime" revision="d935dd05da0cf959e9fd0140af913dc6fdaa0221" upstream="master" dest-branch="master"/>
-  <project name="util_libs.git" path="projects/util_libs" remote="sel4devkit" revision="master" upstream="master" dest-branch="master"/>
+  <project name="util_libs.git" path="projects/util_libs" remote="seL4" revision="73d4ccaeca68dcda83eda49245d774cab73480c0" upstream="master" dest-branch="master"/>
```

Comment:
The comparison url for this example is
https://github.com/sel4devkit/util_libs/compare/master..seL4:util_libs:73d4ccaeca68dcda83eda49245d774cab73480c0
again where green pluses signify lines that are in the Foundation repo, not our local one, with this ordering.

Reply:
There are no significant differences here, and the fresh builds referred to above will confirm - TBD.